src/Tribe/Views/V2/Widgets/List_Widget.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * List Widget
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ * @since   TBD
+ */
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Events\Pro\Views\V2\Assets as Pro_Assets;
+use Tribe\Events\Pro\Views\V2\Shortcodes\Tribe_Events;
+use Tribe\Events\Views\V2\Assets as Event_Assets;
+use Tribe\Events\Views\V2\Manager as Views_Manager;
+use Tribe\Events\Views\V2\Theme_Compatibility;
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\View_Interface;
+use Tribe\Widget\Widget_Abstract;
+use Tribe__Context as Context;
+use Tribe__Events__Main as TEC;
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class for List Widget.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class List_Widget extends Widget_Abstract {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $slug = 'tribe_events_list_widget';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $default_arguments = [
+		// General widget properties
+		'title'         => '',
+
+		// View options
+		'view'                 => null,
+		'should_manage_url'    => false,
+
+		// Event widget options
+		'id'                   => null,
+		'alias-slugs'          => null,
+		'limit'                => 5,
+		'no_upcoming_events'   => '',
+		'featured_events_only' => false,
+		'tribe_is_list_widget' => true,
+		'jsonld_enable'        => true,
+
+		// WP_Widget properties
+		'id_base'              => 'tribe-events-list-widget',
+		'name'                 => null,
+		'widget_options'       => [
+			'classname'   => 'tribe-events-list-widget',
+			'description' => null
+		],
+		'control_options'      => [
+			'id_base' => 'tribe-events-list-widget'
+		],
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: d58e1c1e5ddd28f9a868e16a1b811295</sub>